package.json

```diff
@@ -22,18 +22,16 @@
   },
   "license": "AGPL-3.0",
   "dependencies": {
-    "@edx/frontend-auth": "5.3.6",
-    "@edx/frontend-component-footer": "4.1.5",
-    "@edx/frontend-logging": "3.0.1",
+    "@edx/frontend-component-footer-edx": "3.0.8",
```

Comment:
On the advice of fedx, we're hardcoding the edx version for now (we currently hardcode the edx CSS theme already). This is because fedx does not like the current way of switching between the two footers for MFEs (which involves build time swapping out of npm dependencies, so it's hard to test with the final product).

When we make this repo more open edx friendly, we'll have to revisit the question of theming.